opm/autodiff/FlowMain.hpp

```diff
@@ -141,12 +141,12 @@ namespace Opm
                 return EXIT_FAILURE;
             }
             asImpl().setupOutput();
-            asImpl().setupLogging();
```

Comment:
I can see why you want to call setupLogging() after readDeckInput (since you want to query the MESSAGES keyword). But I don't see why it's moved after setupGridAndProps() and setupState()? Those functions can create log messages themselves, which now will disappear totally.